feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SearchFragment.kt

```diff
@@ -127,6 +131,17 @@ class SearchFragment : DaggerFragment() {
             searchView.setOnCloseListener { false }
         }
     }
+
+    override fun onPause() {
+        super.onPause()
+
+        val imm = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
I'm very sorry, I've forgotten this api requires Android M... so `ContextCompat.getSystemService(requireContext(), InputMethodService::class.java)` looks better. 🙇